lsst/ts/laser/ascii.py

```diff
@@ -227,9 +231,12 @@ def set_register_value(self, set_value):
         """
         if not self.read_only:
             if not self.simulation_mode:
-                message = self.create_set_message(set_value)
-                self.log.debug("sending message to serial port.")
-                self.port.perform_magic(message)
+                try:
+                    message = self.create_set_message(set_value)
+                    self.log.debug("sending message to serial port.")
+                    self.port.perform_magic(message)
+                except TimeoutError as te:
+                    self.log.error(te)
```

Comment:
If you use `log.exception` instead of `log.error` it will print the traceback, which gives more information. Also, are you sure just logging a timeout error is the best way to deal with this? Shouldn't this exception be raised as it was before? This is done at a really low level inside the code.